upsidedown/theme.json

```diff
@@ -0,0 +1,461 @@
+{
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"custom": true,
+			"customGradient": true,
+			"palette": [
+				{
+					"color": "#111111",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#111111",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#111111",
+					"name": "Foreground",
+					"slug": "foreground"
+				},
+				{
+					"color": "#f9f9f9",
+					"name": "Background",
+					"slug": "background"
+				},
+				{
+					"color": "#f9f9f9",
+					"name": "Tertiary",
+					"slug": "tertiary"
```

Comment:
`tertiary` doesn't seem to be used anywhere, so I don't think we need to define it.

Also, `primary`, `secondary`, and `foreground` are using the same colour definition. I think we only need to define this colour once.